lib/graphql/persisted_queries/error_handlers.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require "graphql/persisted_queries/error_handlers/base_error_handler"
+require "graphql/persisted_queries/error_handlers/default_error_handler"
+
+module GraphQL
+  module PersistedQueries
+    # Contains factory methods for error handlers
+    module ErrorHandlers
+      def self.build(handler, options = nil)
+        if handler.is_a?(ErrorHandlers::BaseErrorHandler)
+          handler
+        else
+          build_by_name(handler, options)
+        end
+      end
+
+      def self.build_by_name(name, options)
```

Comment:
Probably the time has come to make an util method and reuse it here and in the `StoreAdapters` module 🙂

Reply:
I took a swing at this.  There's not too much to abstract away, but there's less duplication now.  Let me know what you think or if I missed the point.